lib/prelude.dx

```diff
@@ -2387,6 +2389,25 @@ def throw {a} (_:Unit) : {Except} a =
 def assert (b:Bool) : {Except} Unit =
   if not b then throw ()
 
+'### Misc instances that require `todo`
```

Comment:
This comment is not still true, is it?  There's no `todo` here now, right?

Reply:
Fixed, thanks.